app/src/main/java/org/mozilla/fenix/settings/logins/fragment/LoginDetailFragment.kt

```diff
@@ -206,33 +221,6 @@ class LoginDetailFragment : SecureFragment(R.layout.fragment_login_detail), Menu
         }
     }
 
-    /**
-     * Click listener for a textview's copy button.
-     * @param value Value to be copied
-     * @param snackbarText Text to display in snackbar after copying.
-     */
-    private inner class CopyButtonListener(
-        private val value: String?,
-        @StringRes private val snackbarText: Int,
-    ) : View.OnClickListener {
-        override fun onClick(view: View) {
-            val clipboard = view.context.components.clipboardHandler
-            clipboard.text = value
-            showCopiedSnackbar(view.context.getString(snackbarText))
-            Logins.copyLogin.record(NoExtras())
```

Comment:
It looks like we missed moving this metric to the new click listeners. With this I think it could be good to go :)

Reply:
I have added the metric back to the listeners. thanks